justfile

```diff
@@ -105,6 +105,7 @@ _codegen-order-workaround:
 # Checks rust code, fails on warnings on CI
 rust-check: _codegen-order-workaround
     cd "$RUST_WORKSPACE"; \
+    cargo clippy --all-targets --features="storage" --locked; \
```

Comment:
🔧 this is a duplicate of l.110
```suggestion
```

Reply:
:rocket: a notorious "no error, no warning" merge/rebase failure :scream: